ansible/callback_plugins/log_beautifier.py

```diff
@@ -13,6 +14,11 @@ def fixed_dump_results(self, result, indent=None, sort_keys=True, keep_invocatio
     message_dictionary = json.loads(json_message, encoding="utf-8")
     result = ""
     for key, value in message_dictionary.iteritems():
+        try:
+            value = value.replace(re.search('https://(.*)@github.com',unicode(value)).group(1),"***")
+            value = value.replace(re.search('.rosinstall (.*)',unicode(value)).group(1),"***")
```

Comment:
why are you replacing rosinstall file?

Reply:
I am replacing everything that follows ".rosinstall " note the space after .rosinstall. This is because in the logs, I can see .rosinstall username password